src/directives/ng-file-drop.ts

```diff
@@ -122,6 +122,16 @@ export class NgFileDropDirective implements OnChanges, OnInit {
       });
     }
 
+    if (this.options.maxSize > 0) {
+      this.files = this.files.filter(f => {
+        if (f.size <= this.options.maxSize)
```

Comment:
please use brackets here. I recommend to install `tslint` plugin for your editor.